packages/tools/lib/css-processors/css-processor-components.mjs

```diff
@@ -7,7 +7,7 @@ import chokidar from "chokidar";
 import scopeVariables from "./scope-variables.mjs";
 import { writeFileIfChanged, getFileContent } from "./shared.mjs";
 
-const tsMode = process.env.UI5_TS === "true";
+const tsMode = true;
```

Comment:
this should remain, right? used in the scripts, also on the next line.

Reply:
Yes, it was removed to force generation of typescript file while I was testing this step. I reverted it.